content/blog/2021/containerqueries.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Getting Started with Container Queries
+sub: Now is the time to begin experimenting with a long requested layout tool. 
+author: davidh
+date: 2021-04-02
```

Comment:
We should remember to update this again right before publishing. I think we could also feature it as a large card in the resources list.

Reply:
changed the date to April 5th for now